README.md

```diff
@@ -328,141 +331,157 @@ Number          LandroidScheduleSundayDuration          "Duration [%d]"
-                Setpoint item=LandroidScheduleTuesdayStartMinutes minValue=0 maxValue=45 step=15
-                Setpoint item=LandroidScheduleTuesdayDuration minValue=15 maxValue=1425 step=15
-                Switch item=LandroidScheduleTuesdayEdgecut
-            }
-            Frame label="Schedule Wednesday" {
-                Setpoint item=LandroidScheduleWednesdayStartHour minValue=0 maxValue=23 step=1
-                Setpoint item=LandroidScheduleWednesdayStartMinutes minValue=0 maxValue=45 step=15
-                Setpoint item=LandroidScheduleWednesdayDuration minValue=15 maxValue=1425 step=15
-                Switch item=LandroidScheduleWednesdayEdgecut
-            }
-            Frame label="Schedule Thursday" {
-                Setpoint item=LandroidScheduleThursdayStartHour minValue=0 maxValue=23 step=1
-                Setpoint item=LandroidScheduleThursdayStartMinutes minValue=0 maxValue=45 step=15
-                Setpoint item=LandroidScheduleThursdayDuration minValue=15 maxValue=1425 step=15
-                Switch item=LandroidScheduleThursdayEdgecut
-            }
-            Frame label="Schedule Friday" {
-                Setpoint item=LandroidScheduleFridayStartHour minValue=0 maxValue=23 step=1
-                Setpoint item=LandroidScheduleFridayStartMinutes minValue=0 maxValue=45 step=15
-                Setpoint item=LandroidScheduleFridayDuration minValue=15 maxValue=1425 step=15
-                Switch item=LandroidScheduleFridayEdgecut
-            }
-            Frame label="Schedule Saturday" {
-                Setpoint item=LandroidScheduleSaturdayStartHour minValue=0 maxValue=23 step=1
-                Setpoint item=LandroidScheduleSaturdayStartMinutes minValue=0 maxValue=45 step=15
-                Setpoint item=LandroidScheduleSaturdayDuration minValue=15 maxValue=1425 step=15
-                Switch item=LandroidScheduleSaturdayEdgecut
-            }
-            Frame label="Schedule Sunday" {
-                Setpoint item=LandroidScheduleSundayStartHour minValue=0 maxValue=23 step=1
-                Setpoint item=LandroidScheduleSundayStartMinutes minValue=0 maxValue=45 step=15
-                Setpoint item=LandroidScheduleSundayDuration minValue=15 maxValue=1425 step=15
-                Switch item=LandroidScheduleSundayEdgecut
-            }
+	Group item=Shaun icon="landroid" {
+        Frame {
+			Switch item=LandroidAction label="Action" mappings=[START="Start"] visibility=[LandroidStatusCode==0, LandroidStatusCode==1]
+			Switch item=LandroidAction label="Action" mappings=[STOP="Stop",HOME="Home"] visibility=[LandroidStatusCode==7]
+			Switch item=LandroidAction label="Action" mappings=[STOP="Stop",HOME="Home"] visibility=[LandroidStatusCode==33]
+			Switch item=LandroidAction label="Action" mappings=[START="Start",HOME="Home"] visibility=[LandroidStatusCode==34]
+			Switch item=LandroidPoll label="Refresh" mappings=[ON="Poll"]
```

Comment:
format

Reply:
weird stuff as it looks fine in VSC on my end, will try to update manually on Git

Reply:
solved